### PR TITLE
Reorganize repository 

### PR DESCRIPTION
**Pull request by ThePixelGamer**
This change is [Reviewable](https://reviewable.io/reviews/open-ead/nnheaders/13)

### CODE REVIEW

**Comment by leoetlino** [comment]
On second thought, I think we should have NintendoSDK and NintendoWare folders under modules (and maybe even get rid of the top-level "modules" directory entirely):

* src
  * NintendoSDK
    * nnSdk
    * gfx
    * libcurl
  * NintendoWare
    * g3d
    * atk
    * vfx
    * ui2d
    * font

(inspired by https://discord.com/channels/688807550715560050/794951602396987432/1030560220310671432, but without the top-level 'Nintendo' dir because it's unlikely we'll have Havok in here)

**Comment by ThePixelGamer** [comment]
In that case where should I move nvn?

**Comment by leoetlino** [comment]
NintendoSDK/nvn, probably?

Unrelated, but there's also NintendoSDK/movie

File: CMakeLists.txt [hunk 1]
```diff
@@ -129,7 +129,7 @@ add_library(NintendoSDK OBJECT
   include/vapours/results/sm_results.hpp
   include/vapours/results/psc_results.hpp
 
-  modules/nvn/nvnInit.cpp
+  src/nvn/nvnInit.cpp
```
Comment (leoetlino) [inline CMakeLists.txt:132]:
```suggestion
  src/NintendoSDK/nvn/nvnInit.cpp
```

**Review by leoetlino — CHANGES_REQUESTED** [review]

**Review by leoetlino — APPROVED** [review]

**Pull request merged**